pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -372,8 +372,6 @@ stdenv.mkDerivation {
     + optionalString (libcxx.isLLVM or false) (''
       echo "-isystem ${lib.getDev libcxx}/include/c++/v1" >> $out/nix-support/libcxx-cxxflags
       echo "-stdlib=libc++" >> $out/nix-support/libcxx-ldflags
-    '' + lib.optionalString stdenv.targetPlatform.isLinux ''
```

Comment:
I don't think we always want to remove this, since you are no longer doing the static linking change in this PR?

Reply:
Right. Good catch.